norse/torch/module/izhikevich.py

```diff
@@ -0,0 +1,70 @@
+from typing import Any, Callable, Optional, Tuple
+
+import numpy as np
+import torch
+
+from norse.torch.functional.threshold import threshold
+
+from ..functional.izhikevich import (
+    IzhikevichState,
+    IzhikevichParameters,
+    IzhikevichSpikingBehaviour,
+    izhikevich_step,
+)
+
+from ..functional import izhikevich
+
+from norse.torch.module.snn import SNN, SNNCell, SNNRecurrent, SNNRecurrentCell
+
+
+class IzhikevichCell(SNNCell):
+    """Documentation WIP
+    Module that computes a single Izhikevich neuron-model *without* recurrence and *without* time.
+
+    Parameters:
+        spiking_method (IzhikevichSpikingBehaviour) : parameters and initial state of the neuron
+    Examples:
+        >>> batch_size = 16
```

Comment:
Excellent you added examples! One good practice to follow is to ensure your examples are [SSCCE](http://www.sscce.org/). The only thing you'd need is to add import statements. C.f. my comment about the `__init__` file.